cmd/go-qcow2reader-example/read.go

```diff
@@ -67,7 +67,19 @@ func cmdRead(args []string) error {
 
 	buf := make([]byte, bufferSize)
 	sr := io.NewSectionReader(img, offset, length)
-	_, err = io.CopyBuffer(os.Stdout, sr, buf)
+	w := &Writer{os.Stdout}
+
+	_, err = io.CopyBuffer(w, sr, buf)
 
 	return err
 }
+
+// Writer forces os.File to write using our buffer by hiding it's ReadFrom
+// method. io.CopyBuffer is up to XX times faster with this wrrapper.
+type Writer struct {
```

Comment:
`Writer` isn't a descriptive name.

What about:
```suggestion
type hideReadFrom struct {
```

Reply:
Renamed.